translations/CONTRIBUTING.mal.md

```diff
@@ -0,0 +1,69 @@
+# ഓപ്പൺഇബിഎസിലേക്ക് സംഭാവന ചെയ്യുന്നു
+
+കൊള്ളാം !! കൂടുതൽ ഓപ്പൺഇബിഎസ് ഹാക്കർമാരെ ഞങ്ങൾ എപ്പോഴും അന്വേഷിക്കുന്നു. ഇത് വായിച്ചുകൊണ്ട് നിങ്ങൾക്ക് ആരംഭിക്കാൻ കഴിയും [അവലോകനം](./contribute/design/README.md)
+
+ഒന്നാമതായി, നിങ്ങൾക്ക് എന്തെങ്കിലും ഉറപ്പോ ഭയമോ ഉണ്ടെങ്കിൽ, പ്രശ്നം ചോദിക്കുക അല്ലെങ്കിൽ സമർപ്പിക്കുക അല്ലെങ്കിൽ എന്തായാലും അഭ്യർത്ഥന വലിക്കുക. നിങ്ങളുടെ മികച്ച ശ്രമം നടത്തിയതിന് നിങ്ങൾ ആക്രോശിക്കില്ല. സംഭവിക്കാവുന്ന ഏറ്റവും മോശം കാര്യം എന്തെങ്കിലും മാറ്റാൻ നിങ്ങളോട് മാന്യമായി ആവശ്യപ്പെടും എന്നതാണ്. ഏതെങ്കിലും തരത്തിലുള്ള സംഭാവനകളെ ഞങ്ങൾ അഭിനന്ദിക്കുന്നു, ഒപ്പം നിയമങ്ങളുടെ ഒരു മതിൽ ലഭിക്കാൻ ഞങ്ങൾ ആഗ്രഹിക്കുന്നില്ല.
```

Comment:
Can you fix the grammatical errors in this paragraph?

Also, instead of word by word translation, it would be great if you could translate the underlying meaning of the paragraph.